include/battle.h

```diff
@@ -622,7 +622,7 @@ struct BattleStruct
     struct MegaEvolutionData mega;
     struct ZMoveData zmove;
     const u8 *trainerSlideMsg;
-    bool8 trainerSlideLowHpMsgDone;
+    bool8 trainerSlideLowHpMsgDone:1;
```

Comment:
Why is that, and what do you suggest me to do? :eyes:

Reply:
because of struct alignment. you can't align the next field in the middle of a byte, so it will fill in the remaining 7 bits with padding. you can move the field to another bit-field area. If you don't care about saying the space then its fine as is. I just made this comment in the event you were trying to save the 7 bits

Reply:
I was. I rather remove the bit allocation and keep the variable a `bool8` than to use the u8 data type for what's a bool variable honestly. I'll do that, if you don't mind.